### PR TITLE
[hid] Update NotificationLed structs

### PR DESCRIPTION
**Pull request by CTCaer**

### CODE REVIEW

File: nx/include/switch/services/hidsys.h [hunk 1]
```diff
@@ -11,19 +11,19 @@
...
 
 /// Structure for \ref hidsysSetNotificationLedPattern.
 /// See also: https://switchbrew.org/wiki/HID_services#NotificationLedPattern
 /// Only the low 4bits of each used byte in this struct is used.
 typedef struct {
-    u8 globalMiniCycleDuration;                         ///< Global Mini Cycle Duration. 8ms - 175ms. Value 0x0 = 0ms/OFF.
-    u8 totalMiniCycles;                                 ///< Number of Mini Cycles. 1-15. If number of cycles is > 0 then x0 = x1
-    u8 totalFullCycles;                                 ///< Number of Full Cycles. 1-15. Value 0x0 is repeat forever, but if globalMiniCycleDuration is set to 0x0, it does the 1st Mini Cycle and then the LED stays on with startIntensity.
+    u8 globalMiniCycleDuration;                         ///< Mini Cycle Base Duration. Value 0x1-0xF: 12.5ms - 187.5ms. Value 0x0 = 0ms/OFF.
+    u8 totalMiniCycles;                                 ///< Number of Mini Cycles + 1. Value 0x0-0xF: 1 - 16 mini cycles
```
Comment (yellows8) [inline nx/include/switch/services/hidsys.h]: period

File: nx/include/switch/services/hidsys.h [hunk 2]
```diff
@@ -11,19 +11,19 @@
...
 } HidsysNotificationLedPatternCycle;
 
 /// Structure for \ref hidsysSetNotificationLedPattern.
 /// See also: https://switchbrew.org/wiki/HID_services#NotificationLedPattern
 /// Only the low 4bits of each used byte in this struct is used.
 typedef struct {
-    u8 globalMiniCycleDuration;                         ///< Global Mini Cycle Duration. 8ms - 175ms. Value 0x0 = 0ms/OFF.
-    u8 totalMiniCycles;                                 ///< Number of Mini Cycles. 1-15. If number of cycles is > 0 then x0 = x1
-    u8 totalFullCycles;                                 ///< Number of Full Cycles. 1-15. Value 0x0 is repeat forever, but if globalMiniCycleDuration is set to 0x0, it does the 1st Mini Cycle and then the LED stays on with startIntensity.
+    u8 globalMiniCycleDuration;                         ///< Mini Cycle Base Duration. Value 0x1-0xF: 12.5ms - 187.5ms. Value 0x0 = 0ms/OFF.
```
Comment (yellows8) [inline nx/include/switch/services/hidsys.h]: baseMiniCycleDuration?

Comment (yellows8) [reply nx/include/switch/services/hidsys.h]: (update example pr that afterwards)

Comment (CTCaer) [reply nx/include/switch/services/hidsys.h]: I thought about that and didn't change it because the original also makes sense.
I'll push a fix to both

Comment (CTCaer) [reply nx/include/switch/services/hidsys.h]: done

File: nx/include/switch/services/hidsys.h [hunk 3]
```diff
@@ -12,18 +12,18 @@
...
 } HidsysNotificationLedPatternCycle;
 
 /// Structure for \ref hidsysSetNotificationLedPattern.
 /// See also: https://switchbrew.org/wiki/HID_services#NotificationLedPattern
 /// Only the low 4bits of each used byte in this struct is used.
 typedef struct {
-    u8 globalMiniCycleDuration;                         ///< Mini Cycle Base Duration. Value 0x1-0xF: 12.5ms - 187.5ms. Value 0x0 = 0ms/OFF.
-    u8 totalMiniCycles;                                 ///< Number of Mini Cycles + 1. Value 0x0-0xF: 1 - 16 mini cycles
-    u8 totalFullCycles;                                 ///< Number of Full Cycles. Value 0x1-0xF: 1 - 15 full cycles. Value 0x0 is repeat forever, but if globalMiniCycleDuration is set to 0x0, it does the 1st Mini Cycle with a 12.5ms step duration and then the LED stays on with startIntensity.
+    u8 baseMiniCycleDuration;                         ///< Mini Cycle Base Duration. Value 0x1-0xF: 12.5ms - 187.5ms. Value 0x0 = 0ms/OFF.
```
Comment (yellows8) [inline nx/include/switch/services/hidsys.h]: Comments are misaligned now.

**Pull request merged**